doc/dev/custom_devices.rst

```diff
@@ -153,6 +157,8 @@ After doing so, Catalyst should be able to interface with your custom device.
         pennylane_requires = "0.33.0"
         version = "0.0.1"
         author = "Dummy"
+        author = "Erick Ochoa"
```

Comment:
Do you want to add this here?

Reply:
It is just an example, I don't think it matters. I'll just change it to `An Author`.

```suggestion
        author = "An Author"
```